### ISSUE TITLE
6.3.3 beta InAppBillingSecurity is missing

### ISSUE THREAD

**Issue by xabaddonx1** [jamesmontemagno/InAppBillingPlugin#471]
I have the same issue as [this] (https://github.com/jamesmontemagno/InAppBillingPlugin/issues/322) but at this point the code seems to actually have been removed. This is what I see in InAppBillingImplementation:

![image](https://user-images.githubusercontent.com/46001425/170285178-7abb17de-bfac-4dd6-afa6-32876e5372d4.png)

What is the intended way to Verify purchases on Android now?

**Comment by xabaddonx1** [comment]
Currently I've copied the source that was removed into my code to attempt to verify in the way that was supported before... for some reason it is failing the "if (!sign.Verify(Android.Util.Base64.Decode(signature, 0)))" check though. I've verified that my public key is correct, I am passing purchase.OriginalJson for the signedData and puchase.Signature for the signature. Any ideas why this would not work?
